src/pint/observatory/observatories.py

```diff
@@ -1,970 +1,58 @@
-)
-TopoObs(
-    name="cambridge",
-    aliases=["cam"],
-    itrf_xyz=[3919982.752, 2651.982, 5013849.826],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="princeton",
-    aliases=["princeton"],
-    tempo_code="5",
-    itoa_code="PR",
-    itrf_xyz=[1288748.38, -4694221.77, 4107418.8],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="hamburg",
-    aliases=["hamburg"],
-    itrf_xyz=[3788815.62, 1131748.336, 5035101.19],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="jb_42ft",
-    aliases=["jb42"],
-    itrf_xyz=[3822294.825, -153862.275, 5085987.071],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="jb_mkii",
-    aliases=["jbmk2"],
-    tempo_code="h",
-    itoa_code="J2",
-    itrf_xyz=[3822846.76, -153802.28, 5086285.9],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="jb_mkii_rch",
-    aliases=["jbmk2roach"],
-    itrf_xyz=[3822846.76, -153802.28, 5086285.9],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="jb_mkii_dfb",
-    aliases=["jbmk2dfb"],
-    itrf_xyz=[3822846.76, -153802.28, 5086285.9],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="lwa_sv",
-    aliases=["lwasv"],
-    itoa_code="LS",
-    itrf_xyz=[-1531155.54418, -5045324.30517, 3579583.8945],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="grao",
-    aliases=["grao"],
-    itrf_xyz=[6346273.531, -33779.7127, 634844.9454],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
-TopoObs(
-    name="srt",
-    aliases=["srt"],
-    tempo_code="z",
-    itoa_code="SR",
-    itrf_xyz=[4865182.766, 791922.689, 4035137.174],
-    origin="""Imported from TEMPO2 observatories.dat 2021 June 7.""",
-)
 
-# From Tempo 2021 June 8
-TopoObs(
-    name="quabbin",
-    tempo_code="2",
-    itoa_code="QU",
-    itrf_xyz=[1430913.3496148302, -4495711.383965823, 4278113.974517222],
-    origin="""Imported from TEMPO obsys.dat 2021 June 8.""",
-)
-TopoObs(
-    name="vla_site",
-    tempo_code="c",
-    itoa_code="V2",
-    itrf_xyz=[-1601135.5133304405, -5042005.480977412, 3554875.076856462],
-    origin="""Imported from TEMPO obsys.dat 2021 June 8.""",
-)
-TopoObs(
-    name="gb_20m_xyz",
-    itoa_code="G2",
-    itrf_xyz=[883772.7974, -4924385.5975, 3944042.4991],
-    origin="""Imported from TEMPO obsys.dat 2021 June 8.""",
-)
-TopoObs(
-    name="northern_cross",
-    tempo_code="d",
-    itoa_code="BO",
-    itrf_xyz=[4461242.882451464, 919559.8351226494, 4449633.220012489],
-    origin="""Imported from TEMPO obsys.dat 2021 June 8.""",
-)
+# read the observatories
+read_observatories()
+# potentially override any defined here
+if pint_env_var in os.environ:
```

Comment:
I think it makes sense to put this logic into a `read_observatories_from_usual_location()` function that (for the moment? or not?) gets called on import. Or, since we don't expect users to directly access the observatory registry without going through Observatory methods, just have them call the function without reference to importing anything? Same as the discussion for #1247, `Observatory.get` and `Observatory.names` should cover all the bases.